routes/auth.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const router = require('express').Router();
+const knex = require('../knex.js');
+const boom = require('boom');
```

Comment:
You require boom but never use it. Just delete this line, we don't want boom anyhow. No need to bring in a lib just to create simple err messages.